pkgs/development/python-modules/datacache/default.nix

```
@@ -0,0 +1,27 @@
+{ python, stdenv, buildPythonPackage, fetchPypi
+, pandas, appdirs, progressbar33, requests, typechecks, nose, mock}:
+
+buildPythonPackage rec {
+  version = "1.1.5";
+  pname = "datacache";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b2ca31b2b9d3803a49645ab4f5b30fdd0820e833a81a6952b4ec3a68c8ee24a7";
+  };
+
+  checkInputs = [ ];
+  propagatedBuildInputs = [ pandas appdirs progressbar33 requests typechecks nose mock ];
```

not needed at runtime, should go in checkInputs if anything
```suggestion
  propagatedBuildInputs = [ pandas appdirs progressbar33 requests typechecks mock ];
```